service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -76,7 +80,11 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(
 
         Document document = uploadDocumentService.uploadPDF(userId, authorization, pdf, buildFileName(caseDetails));
 
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+        sdf.setTimeZone(TimeZone.getTimeZone("Europe/London"));
+
         Map<String, Object> data = caseDetails.getData();
+        data.put("dateSubmitted", sdf.format(new Date(System.currentTimeMillis())));
```

Comment:
It would be good to have integration tests confirming that date in the right format is returned from callback if this is business requirement to have date and time of submission captured.

Reply:
CCD requires ISO 8601 format and errors when you try to give it anything that is not this. If we are implementing DateTimeFormatter constants also, are int tests necessary. Personally feel they would be a waste of time. Essentially testing today is today and DateTimeFormatter

Reply:
I agree with the above.

Reply:
Also ccd formats the date and so it appears as it always has done. CCD validate the date going in and in the e2e test we check it is correct. I feel like this does what it needs to do.